streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -387,6 +391,12 @@
                     atLeast(0L),
                     ConfigDef.Importance.LOW,
                     RECONNECT_BACKOFF_MS_DOC)
+            .define(RECONNECT_BACKOFF_MAX_CONFIG,
+                    ConfigDef.Type.LONG,
+                    50L,
```

Comment:
For the streams-broker communications I think it would make sense, yes. We could open a follow up JIRA, probably doesn't need to hold back this KIP.